src/keygen/participant.rs

```diff
@@ -398,8 +399,10 @@ impl KeygenParticipant {
         // message _after_ round one is complete? Likewise for all other rounds.
 
         let message = broadcast_message.into_message(BroadcastTag::KeyGenR1CommitHash)?;
+        let keygen_commit = KeygenCommit::from_message(&message)?;
+        warn!("operating on commit: {:?}", keygen_commit);
```

Comment:
todo: This was added as part of our debug run, I don't think it's helpful to leave it here any longer.